guides/configuration/drupal_application.md

```diff
@@ -29,4 +31,4 @@ crons:
 
 In this example, we are only using the ``mysql`` service. You can use more services (redis, solr...) if you need.
 
-We are also supposing that your Drupal codebase is at the root of your Git repository. If it's not the case, you can change the ``document_root`` to the proper folder where your Drupal files are located.
\ No newline at end of file
+This example presumes that your Drupal codebase is in the "web" directory of your repository. If it's not the case, you can change the ``root`` to the proper folder where your Drupal files are located. Note that putting your Drupal webroot at the root of the repository is not supported.
```

Comment:
Having Drupal `index.php` at `/app` is supported but there will be complicated `web.locations` (see below). I think we should be saying that "is not recommended" instead.

```
locations:
  "/":
    root: ""
    passthru: "index.php"
    allow: false
  "sites/default":
    root: "public/sites/default"
    allow: true
  "sites/all":
    root: "sites/all"
    allow: true
```